api/app.ts

```diff
@@ -1,7 +1,14 @@
 import express from "express";
 import bodyparser from "body-parser";
 import logger from "morgan";
+
+const multer = require('multer');
```

Comment:
should we do something like import * as multer from 'multer' for consistency?
was require giving you an error earlier?

Reply:
`import multer from "multer";` should work

Reply:
The import statement gives me errors so I stuck with require

Reply:
oh you have to `npm install @types/multer` that should solve it.